speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java

```diff
@@ -218,6 +225,7 @@ public static class Builder {
     private Boolean timestamps;
     private Boolean profanityFilter;
     private Boolean smartFormatting;
+    private Long smartFormattingVersion;
```

Comment:
This is a `Long`?

Reply:
Yes this is how other "integer" types are generated in the java sdk

Reply:
Or to be more specific, how params given the type "integer" in the api definitions are generated for the java sdk